src/models/People.js

```diff
@@ -78,7 +78,19 @@ static async displayPeopleProfessional() { // returns array of Timesheet Table
         {name: 'people_id', value:{longValue: userId}},
       ]);
 
-      const capabilities = rawCapabilities.records.map(([{ stringValue: capability }])=> capability);
+      const capabilities = rawCapabilities.records.reduce((acc, curr) => {
+        const [{ stringValue: capability },
+                { stringValue: params1 },
+                { stringValue: params2 },
+                { stringValue: params3 }] = curr;
+
+        return [...acc, 
+                ...((!params1 && !params2 && !params3) ? [capability] : []),
+                ...(params1 ? [`${capability}(${params1})`] : []), 
+                ...(params2 ? [`${capability}(${params2})`] : []), 
+                ...(params3 ? [`${capability}(${params3})`] : [])]
```

Comment:
If params2 is available, the assumption is that the number of parameters is 2 or more.
If params2 is available and params1 is null, then replace it with a `NULL` signifier.

If params3 is available, the assumption is that the number of parameters is 3.
If params1 and/or params2 is null when params3 is available, then we replace them with a 'NULL' signifier.

```suggestion
                ...(params2 ? [`${capability}(${params1 ?? 'NULL'},${params2})`] : []), 
                ...(params3 ? [`${capability}(${params1 ?? 'NULL'},${params2 ?? 'NULL'},${params3})`] : [])]
```